src/test/resources/json/court-application-minimal.json

```diff
@@ -46,6 +46,61 @@
                 "wording": "Has a violent past and fear that he will commit further offences and\n                interfere with witnesse"
               }
             ],
+            "defenceOrganisation": {
```

Comment:
Could we add these changes to a new json file? I believe this one is used to test that we don't throw any validation errors when we get the bare minimum number of expected fields so we want to keep it as sparse as possible.